src/Tools/IdeBenchmarks/IdeBenchmarks.csproj

```diff
@@ -10,6 +10,14 @@
     <!-- Xunit analyzer doesn't recognize benchmark attributes -->
     <NoWarn>$(NoWarn),xUnit1013</NoWarn>
     <UseWpf>true</UseWpf>
+    <!-- This project is not a test project in context of how this repo views test projects.
+         Marking it as such will change TFMs, bring in additional targets, and generally fail to build.
+         However, it is a 'test' type project and depends on other test utility projects that may be built in
+         every build invocation. For intance, when building officially in the VMR, test and test utility projects are excluded.
+         So, when test projects are excluded, exclude this benchmark project.
+         See https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild/Unified-Build-Controls.md for information
+         on controls. -->
+    <ExcludeFromDotNetBuild Condition="'$(DotNetBuildTests)' != 'true'">true</ExcludeFromDotNetBuild>
```

Comment:
Wondering if there should be some `<IsBenchmarkProject>` flag and the logic to exclude those from VMR would be shared somewhere like Directory.Build.props - then we would not need to copy the long comment to each benchmark project.

Reply:
okay, I meant more like a flag only local to the roslyn repo to share this logic instead of copy-pasting it

Reply:
@jjonescz I looked into doing this. There's no place to insert such a flag without duplicating the arcade logic around project exclusion. This is because the common logic that processes project exclusion is imported via a .NET SDK hook prior to any roslyn common targets files (e.g. eng/Imports.BeforeArcade.targets).